patches/chrome-browser-extensions-api-identity-identity_get_auth_token_function.cc.patch

```diff
@@ -103,16 +103,17 @@ index 04a1506295005bed19a905232b07f87fba858c13..456118514ca565d03394201b7d0f7cdc
 +                       " "));
 +  web_auth_flow_ = std::make_unique<WebAuthFlow>(
 +      this, GetProfile(), google_oauth_url,
-+      IsInteractionAllowed(interactivity_status_for_signin_) ?
-+        WebAuthFlow::INTERACTIVE : WebAuthFlow::SILENT,
-+      WebAuthFlow::LAUNCH_WEB_AUTH_FLOW);
++      IsInteractionAllowed(interactivity_status_for_signin_)
++          ? WebAuthFlow::INTERACTIVE
++          : WebAuthFlow::SILENT,
++      WebAuthFlow::LAUNCH_WEB_AUTH_FLOW, extension()->name());
 +  web_auth_flow_->Start();
 +}
 +
  void IdentityGetAuthTokenFunction::OnRefreshTokenUpdatedForAccount(
      const CoreAccountInfo& account_info) {
    if (account_listening_mode_ != AccountListeningMode::kListeningTokens)
-@@ -761,6 +823,84 @@ void IdentityGetAuthTokenFunction::OnGaiaRemoteConsentFlowApproved(
+@@ -761,6 +824,84 @@ void IdentityGetAuthTokenFunction::OnGaiaRemoteConsentFlowApproved(
    StartMintTokenFlow(IdentityMintRequestQueue::MINT_TYPE_NONINTERACTIVE);
  }
  
```

Comment:
this whole `.patch` is `#2` biggest patch across other `brave/patches/*`. not sure why it's like that, but we can create a follow up to fix this.

Reply:
Great point, created https://github.com/brave/brave-browser/issues/28584 to track this issue.